### ISSUE TITLE
Remove WCAG-Scanner violation cc and license

### ISSUE THREAD

**Issue by j-mendez** [danielclough/wtf#1]
The https://github.com/wcat-dev/wcat-scanner   project needs to be removed immediately. It is violating copyright and changes the license violating multiple conditions. Remove the project before people fork it or there will be a DMCA takedown notice that is more permanent. This is the final notice.

**Comment by j-mendez** [comment]
You cannot state project being based on changes without any changes to any of the core. If you’re going to include code you need to respect licensing and not just add a couple files and say brand new project new license. Even if you’re using AI for these things you need to be aware of license and copyright regulations.
